tools/plugins/com.liferay.ide.layouttpl.core/src/com/liferay/ide/layouttpl/core/operation/NewLayoutTplDataModelProvider.java

```diff
@@ -192,6 +192,13 @@ public IStatus validate( String propertyName )
             {
                 return LayoutTplCore.createErrorStatus( Msgs.templateIdInvalid );
             }
+
+            String idText = getStringProperty( propertyName );
+
+            if( "" == idText )
+            {
+                return LayoutTplCore.createErrorStatus( Msgs.templateIdEmpty );
```

Comment:
Just put the string directly here, no need to externalize it, we don't need the Msgs.templateIdEmpty key.